python/pyspark/ml/regression.py

```diff
@@ -1654,7 +1656,7 @@ class _AFTSurvivalRegressionParams(_PredictorParams, HasMaxIter, HasTol, HasFitI
     def __init__(self, *args):
         super(_AFTSurvivalRegressionParams, self).__init__(*args)
         self._setDefault(censorCol="censor",
-                         quantileProbabilities=[0.01, 0.05, 0.1, 0.25, 0.5, 0.75, 0.9, 0.95, 0.99],
+                         quantileProbabilities=DEFAULT_QUANTILE_PROBABILITIES,
```

Comment:
Side note ‒ I remember this was included for Scala parity (SPARK-32310), but isn't that a dead code, give we use the same default in estimator and model constructors?

Reply:
Yes, you are correct. The code isn't being used. I've removed the default argument.

Reply:
Sorry for the confusion ‒ I just wanted to point it out to see if I missed something, but I don't think we should change it here, as this will be a breaking change.

Reply:
I've removed the default argument in the dead code.

Reply:
Yes, since there is only one place where this is used, let's revert this.